glambie/data/timeseries.py

```diff
@@ -334,23 +349,41 @@ def convert_timeseries_to_unit_gt(self, include_area_change: bool = True,
 
         if self.unit == "gt":  # no conversion needed as already in gt
             return copy.deepcopy(self)
-        elif self.unit == "mwe":  # @TODO: convert uncertainties as well
+        elif self.unit == "mwe":
             if not include_area_change:
                 object_copy.data.changes = np.array(meters_water_equivalent_to_gigatonnes(
                     self.data.changes, area_km2=glacier_area, density_of_water=density_of_water))
-                return object_copy
+                # variables for uncertainty calculation
+                area_unc = glacier_area * self.region.area_uncertainty_percentage  # use individual glacier unc
```

Comment:
Can you explain or reference what an "uncertainty percentage" is a bit more please? If it's something close to a hand-wave, we need to say so.